_posts/2023-12-23-welcome.md

```diff
@@ -12,3 +12,8 @@ categories: compiler
 ## Hello world, we are starting a blog.
 
 In the near future, you’ll be seeing long- and short-form blog posts from us so you can get to know us better and read about things we’ve learned, the type of work we do, transit adventures we’ve had, and more! We hope this gives you a useful glimpse into us as a company. See you at our next post.
+
+<figure>
+    <img src="/assets/blog/may2023-retreat.jpg" alt="Photograph of the Compiler team on the 6th Street Bridge in Downtown Los Angeles during their May 2023 Retreat." />
+    <figcaption>The Compiler team on the 6th Street Bridge in Downtown Los Angeles during their May 2023 Retreat. Photograph by [Lex Ryan](https://www.lexryan.co/)</figcaption>
```

Comment:
```suggestion
    <figcaption> {{ "The Compiler team on the 6th Street Bridge in Downtown Los Angeles during their May 2023 Retreat. Photograph by [Lex Ryan](https://www.lexryan.co/)" | markdownify }}</figcaption>
```

I noticed Jekyll didn't process your Markdown that's inside of some HTML. :/

![image](https://github.com/compilerla/compiler.la/assets/25497886/6e7f2c7e-7778-4c66-9db6-d1b98489451e)


I found a [StackOverflow question](https://stackoverflow.com/questions/15917463/embedding-markdown-in-jekyll-html) talking about this problem, and one easy solution is to wrap the contents in a `markdownify` filter, which is what the suggestion above does.